server/model/model.js

```diff
@@ -0,0 +1,67 @@
+const db = require('../../database/index.js');
+const Sequelize = require('sequelize');
+
+const Listing = db.define('listing', {
+  id: {
+    type: Sequelize.INTEGER,
+    primaryKey: true,
+    autoIncrement: true
+  },
+  room_type: {
+    type: Sequelize.TEXT
+  },
+  user_name: {
+    type: Sequelize.TEXT
+  },
+  room_type_details: {
+    type: Sequelize.TEXT
+  },
+  city: {
+    type: Sequelize.TEXT
+  },
+  city_details: {
+    type: Sequelize.TEXT
+  },
+  listing_details: {
+    type: Sequelize.TEXT
+  },
+  guest_access_details: {
+    type: Sequelize.TEXT
+  },
+  interaction_guests_details: {
+    type: Sequelize.TEXT
+  },
+  other_details: {
+    type: Sequelize.TEXT
+  },
+  avatar: {
+    type: Sequelize.TEXT
+  }
+});
+
+const getListing = function(id) {
+  return Listing.findAll({
```

Comment:
Do you need findAll here? If you're only looking for one record, it would be more efficient to use a Sequelize method that stops after finding one record, if that method exists.

Reply:
@wyatt-troia, good catch. All of the listings will be unique, therefore I will only need to find one. I'll update this now and submit another request.

Reply:
@wyatt-troia, I added a commit with the changes listed above. Let me know what you think.